change/@ni-nimble-angular-16d68d67-9ecc-41b3-b7aa-cb52dbd66da4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
FYI JIC handling the intermittent menu stories: https://github.com/ni/nimble/issues/1479#issuecomment-2341882851